pkg/minikube/config/profile.go

```diff
@@ -194,7 +194,7 @@ var DockerContainers = func() ([]string, error) {
 // invalidPs are the profiles that have a directory or config file but not usable
 // invalidPs would be suggested to be deleted
 func ListProfiles(miniHome ...string) (validPs []*Profile, inValidPs []*Profile, err error) {
-
+	activeP := viper.GetString(ProfileName)
```

Comment:
@spowelljr This will create a circular import, should the `ClusterFlagValue()` be moved to some `pkg/util`  as it's more of a utility function.

Reply:
```bash
go: downloading k8s.io/kube-openapi v0.0.0-20220328201542-3ee0da9b0b42
package k8s.io/minikube/cmd/minikube
	imports k8s.io/minikube/cmd/minikube/cmd
	imports k8s.io/minikube/cmd/minikube/cmd/config
	imports k8s.io/minikube/pkg/addons
	imports k8s.io/minikube/pkg/kapi
	imports k8s.io/minikube/pkg/minikube/proxy
	imports k8s.io/minikube/pkg/minikube/config
	imports k8s.io/minikube/cmd/minikube/cmd/config: import cycle not allowed
make: *** [Makefile:215: out/minikube] Error 1
```

Reply:
Sorry, I missed that this file was outside the `cmd` package. That could be something that's done in the future, that flag is used extensively, so that's something that should be in a separate PR as to not bloat this one. I'm happy with this as is.